content/blog/2017-10-31-ozunconf2017.md

```diff
@@ -0,0 +1,115 @@
+
+We wrapped up at the end of day 2, giving each projects group three minutes to debrief on their projects, using the unconf style - only the README.md (mostly!). You can [check out all the ozunconf projects here](https://ropenscilabs.github.io/ozunconf-projects/), thanks to a template from [Sean Kross](http://seankross.com/), soon we'll publish a series of short posts covering all of these great fun projects.
+
+Here's a quick taster:
+
+- [`realtime`](https://github.com/ropenscilabs/realtime). Realtime streamingplots built on the p5.js library.
+
+- [`stow`](https://github.com/ropenscilabs/ozrepro). A simplified version control interface to git, from within R.
+
+- [icon](https://github.com/ropenscilabs/icon). Easily access and insert web icons into HTML and PDF documents.
+
+- [`ochRe`](https://github.com/ropenscilabs/ochRe). Provide Australia-themed Colour Palettes.
+
+We'll share a quick summary of all of the projects over the coming weeks.
+
+### Introducing people to the community.
+
+rOpenSci has had a profound impact on me and my work. At the end of 2015 I got in touch with them to discuss arranging an unconference in Australia, and they welcomed me and my friends. Today, I am proud to be welcoming those from the ozunconf to this big, kind, wonderful community, and say, as Shannon Ellis summed up: ["Hey! You there! You are welcome here"](https://ropensci.org/blog/2017/06/23/community/). It was also really great to have a diverse group of participants at the ozunconf, and in particular, that we had 40% of participants from women or other underrepresented genders.
+
+### Starts, not ends
+
+One thing that I've realised in my involvement with organising and attending these events is that when the unconf ends, it feels a bit sad, sure, to say goodbye to the environment, the community, the friends, and the projects. At the last unconf in LA, we were sending out a stream of tweets, "[it's not](https://twitter.com/nj_tierney/status/868572134548713472) [over](https://twitter.com/MilesMcBain/status/868590677843599360) [until](https://twitter.com/AmeliaMN/status/868605633435533312) [it's](https://twitter.com/MilesMcBain/status/869044724086185985) [over](https://twitter.com/dataandme/status/869664700606406656). But, in reflection, standing back, taking it all in, the unconference doesn't really end - it just begins. It begins many new things - projects, ideas, collaborations, and friendships.
+
+The ozunconf comes to an end. Now, let's get started.
+
+![](https://njtierney.updog.co/img/ozunconf-group-photo.jpg)
+
+### What's Your Story?
+
+A few people have already written about their unconf17 experience. Have you? Share the link in the comments below and we'll add it here.
+
+#### Projects posts
+
+- [FHIR and R: ronfhir](http://www.healthintersections.com.au/?p=2740)
+
+#### Community posts
+
+- [How do you convince others to use R?](https://simplystatistics.org/2017/10/30/how-do-you-convince-others-to-use-r/)
+
+
+# note:
```

Comment:
you could embed the first tweet and list her items as bullets.
or skip the embedding and say
Key #ozunconf lessons from Steph de Silva (link to tweet? or to her twitter)
- The opportunity to try but not succeed is a luxury to be savoured. Mistakes make you a better programmer
- The best thing about R isn't the language, it's the number of people around who want to help
- Your skills are valuable, so your productivity is too. Investing in the tools that maximise it is worthwhile.
- git really is out to get you.

Reply:
This should go before your lovely closing sentiments. So maybe before "Introducing people to the community"

Reply:
OK cool, I've moved this to above "starts, not ends"